webpack.config.js

```diff
@@ -15,27 +13,63 @@ const mediaQueryAliases = {
   '(min-width: 1200px)': 'x-large-and-up',
 };
 
-const entryPoints = blockName => {
-  const tpl = {
-    [`${blockName}EditorScript`]: `./assets/src/blocks/${blockName}/${blockName}EditorScript.js`,
-    [`${blockName}EditorStyle`]: `./assets/src/styles/blocks/${blockName}/${blockName}EditorStyle.scss`,
-    [`${blockName}Script`]: `./assets/src/blocks/${blockName}/${blockName}Script.js`,
-    [`${blockName}Style`]: `./assets/src/styles/blocks/${blockName}/${blockName}Style.scss`,
-  };
-  const pathExists = ([,path]) => fs.existsSync(path);
-  return Object.fromEntries(Object.entries(tpl).filter(pathExists));
+const jsConfig = {
+  ...defaultConfig,
+  output: {
+    filename: '[name].js',
+    path: __dirname + '/assets/build'
```

Comment:
It's missing a `,` at the EOL

Reply:
Well spotted, I had just moved this whole object without looking :sweat_smile:

Reply:
Ah, forgot to push this before merging :facepalm: